x-pack/legacy/plugins/alerting/server/lib/get_create_task_runner_function.ts

```diff
@@ -80,6 +80,13 @@ export function getCreateTaskRunnerFunction({
               return;
             }
 
+            if (
+              alertSavedObject.attributes.throttle &&
+              alertInstance.isThrottled(alertSavedObject.attributes.throttle)
```

Comment:
could probably allow `isThrottled()` to take a null value and return false, to simplify cases like this; ie, remove the `alertSavedObject.attributes.throttle` check.